### DISCUSSION TITLE
Four times as much repo clones as unique cloners per week

### DISCUSSION THREAD

**Discussion by magnetic-eagle** [community/community#21996] [New to GitHub]
<p>I’ve observed in my repo’s Statistics/Traffic page that I had 13 unique cloners but around 50 clones. I don’t understand that - why should everyone clone the repo four times a week ? You normally just clone it ONCE then just do git pull. Are package / branch .zip archives and git pulls counted, too ?</p>
<p>Also, can I see the number of total downloads somewhere (not just the last week) ?</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="TheFGFSEagle" data-post="1" data-topic="245339">
<div class="title">
<div class="quote-controls"></div>
 TheFGFSEagle:</div>
<blockquote>
<p>Also, can I see the number of total downloads somewhere</p>
</blockquote>
</aside>
<p>try this</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://tooomm.github.io/github-release-stats/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181068135-b9d4b838-7887-454c-9b48-894730acd8c0.png" class="site-icon" width="16" height="16">

      <a href="https://tooomm.github.io/github-release-stats/" target="_blank" rel="noopener">tooomm.github.io</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://tooomm.github.io/github-release-stats/" target="_blank" rel="noopener">GitHub Release Stats</a></h3>

  <p>Get the latest release stats like download counts, release dates, author on any GitHub project</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/0/703cd5ac02b18e856cde1a3be6ee1c2ecddc98ce.png" data-download-href="/uploads/short-url/g0TOujU1dSYh3PVBs8FZy7waIX4.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068136-b2eafc2d-6837-40e3-84eb-9dc463f5fa75.png" alt="image" data-base62-sha1="g0TOujU1dSYh3PVBs8FZy7waIX4" width="499" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/0/703cd5ac02b18e856cde1a3be6ee1c2ecddc98ce_2_499x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/0/703cd5ac02b18e856cde1a3be6ee1c2ecddc98ce_2_748x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/0/703cd5ac02b18e856cde1a3be6ee1c2ecddc98ce_2_998x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/0/703cd5ac02b18e856cde1a3be6ee1c2ecddc98ce_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1039×1040 75.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>you must have releases or else it will show, there are no releases for this project</p>

**Comment by kingthorin** [comment, marked as answer]
<ol>
<li>some people don’t understand git.</li>
<li>some people work on multiple systems.</li>
<li>some people work on temporary systems (VMs etc)</li>
<li>your repos might be being cloned in workflows or docker builds?</li>
</ol>

**Comment by magnetic-eagle** [comment]
<aside class="quote no-group" data-username="kingthorin" data-post="3" data-topic="245339">
<div class="title">
<div class="quote-controls"></div>
 kingthorin:</div>
<blockquote>
<p>your repos might be being cloned in workflows or docker builds?</p>
</blockquote>
</aside>
<p>Nope - the specific repo I’m talking about is an aircraft model for a flight simulator. Point 1 could be an explication, point 2 an 3 are unlikely, because usually you don’t install a flight sim on multiple machines. I was just thinking, that maybe git pull’s also count in.</p>

**Comment by magnetic-eagle** [comment]
<p>This looks good - but why does it  say that the release has no download assets available (thus doesn’t show the download count) even though there are two assets listed in the release it links to ? <a href="https://github.com/TheFGFSEagle/c210-family/releases/tag/v1.0.0" class="inline-onebox" rel="noopener nofollow ugc">Release Cessna 210 - the first release ! · TheFGFSEagle/c210-family · GitHub</a></p>

**Comment by jfullstackdev** [comment]
<p>in my opinion, the default assets are not counted,</p>
<p>if a software is being shift, either the author/s will include the <code>.zip</code> of the executable or the executable alone or both, those are counted as the <code>assets</code> and you need to manually add that, just like my example I have the <code>.zip</code> and included in that is the executable</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/4/a4b52d42ddc05ce9d7c06232f8182c2d533ea576.png" data-download-href="/uploads/short-url/nv4tqAWBslyodVGL2GFBS4C6ham.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068138-a678c3d6-bbf8-4d80-8537-5c0483e026f2.png" alt="image" data-base62-sha1="nv4tqAWBslyodVGL2GFBS4C6ham" width="378" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a4b52d42ddc05ce9d7c06232f8182c2d533ea576_2_378x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a4b52d42ddc05ce9d7c06232f8182c2d533ea576_2_567x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/a/4/a4b52d42ddc05ce9d7c06232f8182c2d533ea576.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a4b52d42ddc05ce9d7c06232f8182c2d533ea576_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">670×886 37 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
